xex.cpp

```diff
@@ -1,5 +1,5 @@
 #ifdef IDALDR
-#include "../idaldr.h"
+#include "idaldr.h"// my IDA SDK is setup differently, edit this to point to your header file location
```

Comment:
If this is a specific change to you, could you leave this part of the patch our of the PR? No point breaking previous builds people may have running